platform-configuration/src/main/java/org/codice/ddf/configuration/ConfigurationManager.java

```diff
@@ -33,22 +33,19 @@
  * displayed in the DDF System Settings configuration (but appear in other OSGi
  * bundle configurations such as CXF). These read-only settings are included in
  * the list of configuration settings pushed to registered listeners.
- * 
+ * <p/>
  * Registered listeners implement the ConfigurationWatcher interface and have
  * these DDF configuration settings pushed to them when they come online (aka
  * bind) and when one or more of the settings are changed in the Admin Console.
- * 
  */
 public class ConfigurationManager {
-    private static final Logger logger = LoggerFactory.getLogger(ConfigurationManager.class);
```

Comment:
Do we have a convention for static final fields/constants? Do they have to be all caps?